_posts/2023-12-13-release-0_4_0.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "kroxylicious release 0.4.0"
+date:   2023-12-14 00:00:00 +1300
+author: "Keith Wall"
+author_url: "https://www.github.com/k-wall"
+categories:  releases kroxylcious topic-encryption
+---
+
+The Kroxylicious project is very pleased to announce the [release](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.4.0) of Kroxylicious 0.4.0.
+
+The big news is this release includes our `EnvelopeEncryption` filter which provides an **Encryption-at-Rest** solution for the Apache Kafka(tm) which is _transparent_ to both clients and brokers. We have more work planned to iterate on the implentation to reach production quality. Checkout our [0.5.0 milestone](https://github.com/kroxylicious/kroxylicious/milestones/0.5.0) for details.
```

Comment:
Nits: no "the" before Apache Kafka, and "check out" is always two words unless you're referring to the `git` command
```suggestion
The big news is this release includes our `EnvelopeEncryption` filter which provides an **Encryption-at-Rest** solution for Apache Kafka(tm) which is _transparent_ to both clients and brokers. We have more work planned to iterate on the implentation to reach production quality. Check out our [0.5.0 milestone](https://github.com/kroxylicious/kroxylicious/milestones/0.5.0) for details.
```